src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  if (days >= 7) {
+    finalPrice = 40 * days - 50;
+  } else if (days >= 3) {
+    finalPrice = 40 * days - 20;
+  } else {
+    finalPrice = 40 * days;
+  }
+
+  return finalPrice;
```

Comment:
The function name 'calculateRentalCost' is descriptive enough, but the implementation uses hardcoded values for the daily cost and the discounts. Consider defining these as constants at the top of your function or file to make the code more maintainable and self-explanatory.